app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comments_params)
+    @post = Post.find_by(id: params[:post_id])
+    @comment.author_id = current_user.id
+    @comment.post_id = @post.id
+
+    if @comment.save
+      redirect_to user_post_path(user_id: @post.author_id, id: @post.id)
+    else
+      render :new, alert: ':( Cannot Create comment retry again :('
+    end
+  end
+
+  private
+
+  def comments_params
+    params.require(:comments).permit(:text)
```

Comment:
- [ ] I am unable to create a comment. You should use `comment`, instead of `comments` based on the error message.

  ![ifof](https://user-images.githubusercontent.com/36542986/205068763-55194845-0fd3-47e5-9990-0d0bfc2bc799.jpg)